entity-api-spec.yaml

```diff
@@ -1772,6 +2091,11 @@ paths:
                 type: array
                 items:
                   type: object
+              examples:
+                ancestororganarray:
+                  value: 
+                    - 'abcd1234-ef56-gh78-ij90-klmnop123456' 
+                    - 'abcd1234-ef56-gh78-ij90-klmnop123456'
```

Comment:
The result of `GET /entities/{id}/ancestor-organs` is a json array of organ objects, not just an array of ids.